### DISCUSSION TITLE
Fail to run lammps+kokkos+mace

### DISCUSSION THREAD

**Discussion by liubaoshuai1402** [ACEsuit/mace#745] [Q&A] [answered]
Dear developers,

I have tried to build and install lammps+kokkos+mace in many times and tried different computers.
But when I run the builded LAMMPS, it always reports the same errors.
This problem troubles me for almost one month ! I really hope getting your help !
There are two related log file: 
[slurm-82285.out.txt](https://github.com/user-attachments/files/18093585/slurm-82285.out.txt)
[log.txt](https://github.com/user-attachments/files/18093602/log.txt)

**The core error appearing at slurm.out is followed:**
Exception: **The specified pointer resides on host memory and is not registered with any CUDA device**.
Exception raised from getDeviceFromPtr at /opt/conda/conda-bld/pytorch_1729647429097/work/aten/src/ATen/cuda/CUDADevice.h:16 (most recent call first):
frame #0: c10::Error::Error(c10::SourceLocation, std::string) + 0x96 (0x14c42a599446 in /home-ssd/Users/nsgm_lbs/.conda/envs/python311/lib/python3.11/site-packages/torch/lib/libc10.so)

I really don't know which step goes wrong ! Is I don't activate my CUDA correctly?

Thanks!

**Comment by liubaoshuai1402** [comment, marked as answer]
This error is caused by I have used too large timestep in the input file of lammps. Be careful of the unit !

**Reply by Jack20220830** [reply]
Thank your answer.It is very import to me
